en/developer-docs/docs/develop-components/work-with-the-micro-integrator-runtime-in-choreo.md

```diff
@@ -178,61 +184,43 @@ parameter.connection_factory_type = "queue"
-2. Replace the dot character in the package name with an underscore.
+2. Replace the dot (`.`) characters in the package or class name with an underscores(`_`).
 3. Set the variable value to the required logging level for the corresponding package or class.
 
-    For example, to enable wire logs, change the logging level of the `org.apache.synapse.transport.http.wire` package to `debug`. 
 
-To configure logging, set the environment variable as follows:
+For example, to enable wire logs, change the logging level of the `org.apache.synapse.transport.http.wire` package to `debug`, set the environment variable as follows:
 
-1. In the left navigation menu, click **Deploy** and then click **Configs & Secrets**. 
-2. Click **+ Create**. 
-3. Select **ConfigMap** as the **Config Type** and **Environment Variables** as the **Mount Type**. 
-4. Click **Next**. 
-5. Specify the following values as mount configurations:
+1. Select the MI component from the Choreo Console.
+2. From the left navigation, click **Deploy**.
+3. Click on the **Configure and Deploy**.
+4. In **Configuration** step, click **+ Add** on the **Environment Configurations**
+5. Specify the following values as configurations:
 
     | **Field**       | **Value**                                              |                                    
     |-----------------|--------------------------------------------------------|
-    | **Config Name** | An appropriate name for the configuration.             |
     | **Name**        | `logging_level_org_apache_synapse_transport_http_wire` |
     | **Value**       | `debug`                                                |
 
-6. Click **Create**.
+6. Click **Add**.
+7. Click **Next**.
+8. In **Endpoint Details** step, click **Deploy**.
+
+Once the environment variable is set, the logging level for the specified package or class will be updated accordingly. You can add as many logging configurations as you need by following the same steps.
 
 ## Connectors
 
-WSO2 Micro Integrator (MI) Connectors are prebuilt connectors designed to simplify integration between WSO2 MI and various other systems. They enable seamless connections to databases, message brokers, REST APIs, and more, allowing you to perform actions such as sending messages, executing queries, or retrieving data. These connectors are easy to use, making it straightforward to incorporate them into your WSO2 MI integration flows. Micro Integrator connectors are versatile and can be utilized in various integration scenarios, including data integration, service-oriented architecture (SOA) integration, and event-driven architecture (EDA) integration.
+WSO2 Micro Integrator (MI) Connectors are prebuilt components that simplify integrating MI with various external systems. They enable seamless connections to databases, message brokers, REST APIs, and more, allowing you to perform actions such as sending messages, executing queries, or retrieving data within your integration flows.
```

Comment:
I don't think the word "component" is correct in this context. May be we can say extensions instead.